l1-contracts/test/foundry/l1/unit/concrete/state-transition/ChainTypeManager/FreezeChain.t.sol

```diff
@@ -31,4 +31,24 @@ contract freezeChainTest is ChainTypeManagerTest {
         vm.expectRevert(bytes("q1"));
         isChainFrozen = gettersFacet.isDiamondStorageFrozen();
     }
+
+    function test_RevertWhen_UnfreezingChain() public {
```

Comment:
This test while sort of fine, tests wrong behavior. Basically it tests that unfreezing chain is not possible, which is definitely not something we would like to have as a property of the system :) 

This is the case because in `ChainTypeManagerTest` `facetCuts` array has every facet as freezable, please make Getters and Admin facet not freezable (same as in prod) and make sure that this test checks that the it can be unfrozen

Reply:
Resolved.